maquetado/scss/tech.scss

```diff
@@ -12,41 +11,34 @@
 }
 
 .tc-container {
-    display: flex;
-    flex-wrap: wrap;
     margin-left: 100px;
```

Comment:
The `margin: auto` from bottom overrides this `margin-left` value if this is the case remove this CSS rule.